tests/wallet/test_wallet.py

```diff
@@ -177,6 +177,347 @@ async def test_wallet_reuse_address(
+            wallet_node_2.config["trusted_peers"] = {server_1.node_id.hex(): server_1.node_id.hex()}
+        else:
+            wallet_node.config["trusted_peers"] = {}
+            wallet_node_2.config["trusted_peers"] = {}
+        wallet_node_2.config["auto_claim"] = False
+
+        await server_2.start_client(PeerInfo(self_hostname, uint16(server_1._port)), None)
+        await server_3.start_client(PeerInfo(self_hostname, uint16(server_1._port)), None)
+        expected_confirmed_balance = await full_node_api.farm_blocks_to_wallet(count=num_blocks, wallet=wallet)
+
+        assert await wallet.get_confirmed_balance() == expected_confirmed_balance
+        assert await wallet.get_unconfirmed_balance() == expected_confirmed_balance
+        expected_confirmed_balance = await full_node_api.farm_blocks_to_wallet(count=num_blocks, wallet=wallet_1)
+        normal_puzhash = await wallet_1.get_new_puzzlehash()
+        # Transfer to normal wallet
+        tx = await wallet.generate_signed_transaction(
+            uint64(500),
+            normal_puzhash,
+            uint64(0),
+            puzzle_decorator_override=[{"decorator": "CLAWBACK", "clawback_timelock": 5}],
+        )
+
+        await wallet.push_transaction(tx)
+        await full_node_api.wait_transaction_records_entered_mempool(records=[tx])
+        expected_confirmed_balance += await full_node_api.farm_blocks_to_wallet(count=num_blocks, wallet=wallet)
+        # Check merkle coins
+        await time_out_assert(20, wallet_node.wallet_state_manager.merkle_coin_store.count_small_unspent, 1, 1000)
+        await time_out_assert(20, wallet_node_2.wallet_state_manager.merkle_coin_store.count_small_unspent, 1, 1000)
+        assert await wallet.get_confirmed_balance() == 3999999999500
+        # Reorg before claim
+        # Test Reorg mint
+        height = full_node_api.full_node.blockchain.get_peak_height()
+        if height is None:
+            assert False
+        await full_node_api.reorg_from_index_to_new_index(
+            ReorgProtocol(uint32(height - 2), uint32(height + 1), normal_puzhash, None)
+        )
+        await time_out_assert(20, wallet_node.wallet_state_manager.merkle_coin_store.count_small_unspent, 0, 1000)
+        await time_out_assert(20, wallet_node_2.wallet_state_manager.merkle_coin_store.count_small_unspent, 0, 1000)
+
+        expected_confirmed_balance += await full_node_api.farm_blocks_to_wallet(count=num_blocks, wallet=wallet)
+        # Claim merkle coin
+        await asyncio.sleep(20)
+        expected_confirmed_balance += await full_node_api.farm_blocks_to_wallet(count=num_blocks, wallet=wallet)
+        # clawback merkle coin
+        merkle_coin = tx.additions[0] if tx.additions[0].amount == 500 else tx.additions[1]
+        resp = await api_1.spend_clawback_coins(
+            dict({"merkle_coin_ids": [merkle_coin.name().hex(), normal_puzhash.hex()], "fee": 1000})
+        )
+        json.dumps(resp)
```

Comment:
The api call will not serialize the response. For example, you can set a bytes in the response but it will fail in a real call. This is to capture that type of bug.